gcc/d/patches/patch-versym-os-8.x

```diff
@@ -137,27 +166,14 @@ These official OS versions are not implemented:
 +#define TARGET_OS_D_BUILTINS() 			\
 +  do 						\
 +    {						\
-+      TARGET_GENERIC_LINUX_OS_D_BUILTINS();	\
++      GNU_USER_TARGET_OS_D_BUILTINS();		\
 +      ANDROID_TARGET_OS_D_BUILTINS();		\
 +    }						\
 +  while (0)
 +
  #undef CC1_SPEC
  #define CC1_SPEC \
    LINUX_OR_ANDROID_CC (GNU_USER_TARGET_CC1_SPEC, \
---- a/gcc/config/i386/mingw-w64.h
-+++ b/gcc/config/i386/mingw-w64.h
-@@ -89,3 +89,10 @@ along with GCC; see the file COPYING3.  If not see
-   %{static:-Bstatic} %{!static:-Bdynamic} \
-   %{shared|mdll: " SUB_LINK_ENTRY " --enable-auto-image-base} \
-   %(shared_libgcc_undefs)"
-+
-+#undef TARGET_OS_D_BUILTINS
-+#define TARGET_OS_D_BUILTINS()					\
-+    do {							\
-+      TARGET_GENERIC_MINGW_OS_D_BUILTINS();			\
-+      builtin_define ("GNU_MinGW64");				\
-+    } while (0)
 --- a/gcc/config/i386/mingw32.h
```

Comment:
IIRC mingw-w64 also supports 32bit code but I don't think the difference mingw-w64@32bit vs mingw@32 bit matters for druntime.

https://en.wikipedia.org/wiki/MinGW#MinGW-w64